src/Home/Footer.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styles from "./style.module.css";
+import { footerLinks } from "../constants";
+
+const Footer = () => {
+  return (
+    <footer className={styles.footer}>
+      <div className={styles.footerColumns}>
+        {footerLinks.map((category) => (
+          <ul className={styles.footerLinks} key={category.title}>
```

Comment:
is `category.title` a unique and stable constant here. If not, what other alternatives can we have here?